docker-compose.prod.yml

```diff
@@ -16,6 +16,8 @@ services:
       - ANSWER_THREADS=8
       - BATCH_SIZE=672
       - FLASK_ENV=production
+    labels:
+      - logio=yes
```

Comment:
Maybe we also need logs from another container like elastic's? Or you think it isn't necessary?

Reply:
Yes, we can add all the containers that we want. In my local tests, the elastic's log didn't give me relevant information.

Reply:
Okay. Then it's fine.